eng/testing/tests.singlefile.targets

```diff
@@ -34,6 +39,10 @@
     <SelfContained>true</SelfContained>
   </PropertyGroup>
 
+  <PropertyGroup Condition="">
+    <DefineConstants>$(DefineConstants);SINGLE_FILE_TEST_RUNNER</DefineConstants>
+  </PropertyGroup>
```

Comment:
I don't understand this change. We had this line at the top of the file. Now it's moved here with an empty `Condition=""` - why?

Reply:
Oops for some reason, that condition wasn't copied. It has to do with Single File and NativeAOT. Will add it on the next iteration. Thanks for catching this.